test/parallel/test-https-unix-socket-self-signed.js

```diff
@@ -9,8 +9,8 @@ common.refreshTmpDir();
 const fs = require('fs');
 const https = require('https');
 const options = {
-  cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
-  key: fs.readFileSync(common.fixturesDir + '/test_key.pem')
+  cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`),
```

Comment:
@zyziyun Would you consider altering these to use `path.join()` instead?  For example:

```js
  cert: fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'),
```